TrackletAlgorithm/Constants.h

```diff
@@ -99,6 +101,7 @@ constexpr double kz0 = 0.0585938;
 
 // layer projection digitization constants
 constexpr double kphiproj456 = 7.71867e-06;
+constexpr double kphiderdisk = kphi / kr / 128;
```

Comment:
Where does 128 come from? (But I see there are many hardcoded numbers in this file...)